### PR TITLE
Add running of the REST TCK as an option

### PR DESCRIPTION
**Pull request by arjantijms**
Signed-off-by: arjantijms <arjan.tijms@gmail.com>

### CODE REVIEW

File: appserver/tests/tck/rest/pom.xml [hunk 1]
```diff
@@ -0,0 +1,305 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2021, 20202 Contributors to the Eclipse Foundation. All rights reserved.
```
Comment (gurunrao) [inline appserver/tests/tck/rest/pom.xml:4]: copyright typos.

File: appserver/tests/tck/rest/pom.xml [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+        <artifactId>tck</artifactId>
+        <version>6.2.4-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>glassfish-external-tck-rest</artifactId>
+    <packaging>pom</packaging>
+
+    <name>TCK: REST</name>
+
+    <properties>
```
Comment (gurunrao) [inline appserver/tests/tck/rest/pom.xml:34]: migrated JAXRS TCK(https://github.com/eclipse-ee4j/jaxrs-api/tree/master/jaxrs-tck) will not have ts.jte and will also have more tests to run.

Comment (arjantijms) [reply appserver/tests/tck/rest/pom.xml:34]: Yes indeed, so the version of REST in master will look quite a bit different. Same will hold for Faces and maybe Authentication (a Maven based TCK for Authentication is on the roadmap).

File: appserver/tests/tck/rest/pom.xml (hunk 1 above)
Comment (arjantijms) [reply appserver/tests/tck/rest/pom.xml:4]: Thanks!

**Pull request merged**